addons/tracing/tracing.go

```diff
@@ -27,13 +27,14 @@ import (
 	"github.com/apache/camel-k/pkg/util"
 )
 
-// The Tracing trait can be used to automatically publish tracing information to an
-// OpenTracing compatible collector.
+// The Tracing trait can be used to automatically publish tracing information to an OpenTracing compatible collector.
 //
 // The trait is able to automatically discover the tracing endpoint available in the namespace (supports **Jaeger**).
 //
 // The Tracing trait is disabled by default.
 //
+// WARNING: The Tracing trait can't be enabled at the same time as the Telemetry trait.
```

Comment:
I'd take the opportunity and add a deprecation notice at least here, reminding to use Telemetry trait.